server/integration/.editorconfig

```diff
@@ -0,0 +1,14 @@
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
```

Comment:
We either should provide `.editorconfig` for whole Infinispan project or remove this file. My vote would be for the former.

Reply:
It shouldn't have been in this PR so I'll remove it.